mmv1/templates/terraform/constants/network_firewall_policy_rule.go.tmpl

```diff
@@ -0,0 +1,8 @@
+// Match a string that starts with one / slash followed .* t then suppress it as API takes // or / as prefixes
+func SecurityProfileGroupPrefixSlashes(_, old, new string, _ *schema.ResourceData) bool {
+	re := regexp.MustCompile(`^/[^/].*`)
+	if re.MatchString(new) {
+		return true
```

Comment:
From the bug report and the comment it looks like the issue is that the API accepts values that start with a single or double slash, but will always return a double slash. If that's the case, I think the logic here is wrong, as it will always suppress the diff if the new value begins with a single slash (which prevents changing the value from one single-leading slash value to a different single slash value).

A common approach in diffSuppressFuncs is to normalize the old and values and then compare; in this case you could make them both single slash values and compare, something like:

```
if strings.HasPrefix(old, "//") {
    old = old[1:]
}
if strings.HasPrefix(new, "//") {
    new = new[1:]
}
return old == new
```